rb/.ruby-version

```diff
@@ -1 +1 @@
-3.0.6
+jruby-9.4.5.0
```

Comment:
is there a way to set this just for RBE? after working with both for this release, using 3.0.6 ended up being much easier.

Reply:
I think I was running into problems using jruby with RubyMine, though I don't remember exactly what my problem was now, I kept doing and undoing things so often. :)

Reply:
No, I've reverted this.